src/strings/ops.c

```diff
@@ -1,6 +1,7 @@
 #include "platform/memmem.h"
 #include "moar.h"
 #define MVM_DEBUG_STRANDS 0
+#define MAX_GRAPHEMES     0xFFFFFFFF
```

Comment:
Also how do you feel about naming this MAX_STRING_INDEX instead of MAX_GRAPHEMES?